services/cloudfront/src/main/java/software/amazon/awssdk/services/cloudfront/auth/Asn1Object.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.cloudfront.auth;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+
+@SdkInternalApi
```

Comment:
Put `SdkInternalApi` classes in the `internal` package like /internal/auth
If there are auth classes that *need* to be public, keep them in /auth

Reply:
Created internal package and moved files